### PR TITLE
add Datree in Security and PKI

### PR DESCRIPTION
**Pull request by eyarz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,6 +520,7 @@ Table of Contents
   * [Cmd](https://cmd.com/) — Security platform providing real-time access control and dynamic policy enforcement on every Linux instance in your cloud or datacenter
   * [CodeNotary.io](https://www.codenotary.io/) — Open Source platform with indelible proof to notarize code, files, directories or container
   * [crypteron.com](https://www.crypteron.com/) — Cloud-first, developer-friendly security platform prevents data breaches in .NET and Java applications
+  * [Datree](https://www.datree.io/) — Open Source CLI tool to prevent Kubernetes misconfigurations by ensuring that manifests and Helm charts follow best practices as well as your organization’s policies
```
Comment (ripienaar) [inline README.md:523]: please add something like free for up to 1000 policy checks a month.

**Comment by eyarz** [comment]
@ripienaar the pricing is going to be changed in a week or two, so it will be unlimited policy checks.
Do you prefer to wait until it will be updated?

**Comment by ripienaar** [comment]
Yes. Thanks please send an update once the new pricing is out

**Comment by eyarz** [comment]
NP
in the meanwhile, can I leave this PR open?

**Comment by ripienaar** [comment]
yeah no problem

**Comment by eyarz** [comment]
@ripienaar New year, new pricing - [free forever](https://www.datree.io/pricing)! 🥳
image

File: README.md (hunk 1 above)
Comment (eyarz) [reply README.md:523]: new pricing doesn't include policy checks limit.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**